trust-assumptions/mainnet-trust-disclosure-provider-answers.md

```diff
@@ -63,13 +63,13 @@ The Threshold Council may be expanded, discontinued or have the scope of its pow
 #### How does the user-facing client connect to the infrastructure provider’s network? How does it connect to other layers?
 
 _Genesis_ \
-The primary conduit to the Threshold network to access TACo services is via [_Porter_](https://docs.threshold.network/app-development/threshold-access-control-tac/porter)_:_&#x20;
+The primary conduit to the Threshold network to access TACo services is via [_Porter_](../architecture/porter.md)_:_&#x20;
 
 * Porter is an intermediary gateway to the Threshold network. Adopters specify a Porter URI of their choice.&#x20;
 * Porter cannot decrypt private data, as the data remains end-to-end encrypted through the Porter stage of an access request. However, Porter can theoretically block requests.&#x20;
 * The NuCypher team runs a permanent Porter instance, but it is entirely [open source](https://github.com/nucypher/nucypher-porter). Threshold encourages TACo's adopting developers to spin up their own Porter, if that improves the trust burden – this depends on the unique trust assumptions of their use case(s).&#x20;
 
-The default API is a web-oriented API written in Typescript ([taco-ts](https://github.com/nucypher/nucypher-ts)). There is also a reference Python API.&#x20;
+The default API is a web-oriented API written in Typescript ([taco-ts](https://github.com/nucypher/taco-web)). There is also a reference Python API.&#x20;
```

Comment:
Does it make sense to remove the Python API since we no longer document it, and ideally want adopters using `taco-web`?

```suggestion
The default API is a web-oriented API written in Typescript ([taco-ts](https://github.com/nucypher/taco-web)). 
```